src/test/ui/bad/bad-mid-path-type-params.stderr

```diff
@@ -1,26 +1,26 @@
-error[E0087]: too many type parameters provided: expected at most 1 type parameter, found 2 type parameters
-  --> $DIR/bad-mid-path-type-params.rs:40:28
+error[E0087]: wrong number of type arguments: expected 1, found 2
+  --> $DIR/bad-mid-path-type-params.rs:40:13
    |
 LL |     let _ = S::new::<isize,f64>(1, 1.0);
-   |                            ^^^ expected 1 type parameter
+   |             ^^^^^^^^^^^^^^^^^^^ unexpected type argument
```

Comment:
Hmm, this seems to have regressed, could you somehow use each `GenericArg`'s individual span?